{{ cookiecutter.municipality_slug }}/python/example_scraper.py

```diff
@@ -0,0 +1,133 @@
+            EventMinutesItem(
+                minutes_item=MinutesItem(
+                    name=f"Example Minutes Item {random.randint(1, 100)}",
+                    description="Example Description",
+                ),
+                matter=Matter(
+                    name=f"Example Matter {random.randint(1, 100)}",
+                    matter_type=f"Example Matter Type {random.randint(1, 5)}",
+                    title="Example Matter Title",
+                    sponsors=[
+                        _get_example_person(seat_num) for seat_num in sponsors_seats[i]
+                    ],
+                ),
+                supporting_files=[
+                    SupportingFile(
+                        name=f"Example Supporting File Name {file_num}",
+                        uri="https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf",
+                    )
+                    for file_num in range(1, random.randint(1, 5) + 1)
+                ],
+                decision=f"Example Decision {random.randint(1, 5)}",
+                votes=[
+                    Vote(
+                        person=_get_example_person(seat_num),
+                        decision=f"Example Decision {random.randint(1, 5)}",
+                    )
+                    for seat_num in range(1, FAKE_COUNCIL_SEAT_NUM + 1)
+                ],
+            )
+            for i in range(event_minutes_items_num)
+        ]
+        # Insert a non-matter event minutes item
+        event_minutes_items.insert(
+            0,
+            EventMinutesItem(
+                minutes_item=MinutesItem(
+                    name="Example Minutes Item", description="Example Description"
+                )
+            ),
+        )
+    return EventIngestionModel(
+        body=body,
+        sessions=sessions,
+        event_minutes_items=event_minutes_items,
+        agenda_uri="https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf",
+        minutes_uri="https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf",
+    )
+
+
+def _get_example_person(seat_num: int) -> Person:
+    "Create a fake example person"
+    return Person(
+        name=f"Example Person {seat_num}",
+        email="person@example.com",
+        phone="123-456-7890",
+        website="www.example.com",
+        picture_uri="https://councildataproject.github.io/imgs/public-speaker-light-purple.svg",
+        seat=Seat(
+            name=f"Example Seat Position {seat_num}",
+            electoral_area=f"Example Electoral Area {seat_num}",
+            electoral_type=f"Example Electoral Type {1 if seat_num <= FAKE_COUNCIL_SEAT_NUM / 2 else 2 }",
+            image_uri="https://councildataproject.github.io/imgs/seattle.jpg",
+        ),
+        roles=[
+            Role(title="Councilmember", body=Body(name="Example Committee")),
+            Role(title="Chair", body=Body(name=f"Example Committee {seat_num}")),
+        ],
```

Comment:
Maybe if the `seat_num` is 1 they should also be council president?

Reply:
The `seat_num` is based on district areas like District 1 or Citywide position. I think any of the seat positions can be president right?

Reply:
Totally agree. I was just offering a method to select a council present.

Reply:
In a consistent manner between runs that is.

Reply:
Maybe I can add `Role` council president for Example Person 1?

Reply:
Yea. Works for me.